app/(auth)/login/page.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+import Button from '@/components/Button';
+import handleGoogleSignIn from '@/utils/google-ver';
+import LoginForm from '@/components/LoginForm';
+import { useRouter } from 'next/navigation';
+import { getAuth, onAuthStateChanged, User } from 'firebase/auth';
+import app from '@/firebaseconfig';
+
+function Page() {
+  const router = useRouter();
+  const auth = getAuth(app);
+
+  const monitorAuthState = async () => {
+    onAuthStateChanged(auth, (user: User | null) => {
+      console.log(user);
```

Comment:
I think you should remove the console.log here, we wouldn't want our users to see this, you could look into [PRODUCTION and PREVIEW build](https://vercel.com/docs/concepts/deployments/environments) and potentially only log if it's a preview build or simply add it for development purposes.